docs/2/installer/lorawan/gateway-installations/outdoor-istation.md

```diff
@@ -125,4 +132,7 @@ Once all secure, turn on the power of the PoE injector. You can now start deploy
 
 You’re all done! Please send an email to support@microshare.io to indicate the Gateway is up and running so we can confirm at our end.
 
- 
\ No newline at end of file
+---------------------------------------
+
+### Reminder: for optimal performances
+{% include image.html width=400 url="/assets/img/LoRaWAN Device pictures/Without text.jpg" description="LoRaWan Technology" %}
```

Comment:
no space, it should be dashes and lower case. Take a look at the other images in the same page